core/state/statedb.go

```diff
@@ -256,6 +256,32 @@ func (self *StateDB) GetState(addr common.Address, hash common.Hash) common.Hash
 	return common.Hash{}
 }
 
+// MerkleProof
+type ProofList [][]byte
+
+func (n *ProofList) Put(key []byte, value []byte) error {
+	*n = append(*n, value)
+	return nil
+}
+
+// returns the MerkleProof for a given Account
+func (self *StateDB) GetProof(a common.Address) [][]byte {
```

Comment:
Also, a docstring should be on the form
```
// GetProof returns the MerkleProof for a given Account
```
And all public (upper-case) methods should have docstrings

Reply:
thanks, fixed